src/contexts/UserContext.js

```diff
@@ -51,15 +56,19 @@ class UserProvider extends React.Component {
       .then((res) => {
         console.log(res);
         const userReturn = res.data.user.username;
-        const userToken = res.data.user.token;
+        const userToken = res.data.token;
         this.setState({ currentUser: userReturn, token: userToken });
+        localStorage.setItem("currentUser", userReturn);
```

Comment:
Similar to the url building module we discussed, I would create a storage module and only access the `localStorage` global object there. That will centralize the dependency, make it clear at a glance what apis we depend on, and probably make it easier to mock while testing.

Reply:
Are you saying that I should add the data to the local storage as an object?